src/element-stub.js

```diff
@@ -24,7 +24,7 @@ export class ElementStub extends BaseElement {
   constructor(element) {
     super(element);
     // Fetch amp-ad script if it is not present.
-    insertAmpExtensionScript(this.getWin(), element, 'amp-ad');
+    insertAmpExtensionScript(this.getWin(), 'amp-ad');
```

Comment:
Can we call with this element's `#tagName` instead? It should (essentially) be a noop if it's not an `amp-ad` or `amp-embed`, and those two will only install the extension script once.